tests/e2e/__testers__.ts

```diff
@@ -4,29 +4,49 @@ import { values } from 'lodash'
 import stripAnsi from 'strip-ansi'
 import { ClientBase } from '@prisma-spectrum/reflector/dist-cjs/Client'
 import { Reflector } from '@prisma-spectrum/reflector'
+import { getPrismaClient } from '~/tests/e2e/helpers/getPostgresAdminPrismaClient'
+import { PrismaClient } from '@prisma/client'
 
-export const testTemplate = (params: {
+export interface DBTestParams {
   templateName: PrismaTemplates.$Types.Template['_tag']
-  /** A string or regex pattern of the expected output (stdout) when the project dev mode is run.*/
+  testName: string
```

Comment:
Consumer should not have to manage this. Test function has enough context to automate it.

Reply:
I thought being specific would bring more clarity around the test cases since dynamic test names could potentially confuse devs.

I updated the PR as per review.